scene/2d/camera_2d.cpp

```diff
@@ -169,7 +207,7 @@ Transform2D Camera2D::get_camera_transform() {
 		Point2 screen_offset = (anchor_mode == ANCHOR_MODE_DRAG_CENTER ? (screen_size * 0.5 * zoom_scale) : Point2());
 		Rect2 screen_rect(-screen_offset + camera_pos, screen_size * zoom_scale);
 
-		if (limit_smoothing_enabled) {
+		if (limit_smoothing_enabled && limit_enabled) {
```

Comment:
Nitpicking / not really an issue, but I'd say it makes more sense to put more general conditions first:
```suggestion
		if (limit_enabled && limit_smoothing_enabled) {
```
(same for other similar checks)

Reply:
> (same for other similar checks)

You've changed this specific check. :upside_down_face: